src/variables/components/TypeAheadVariableDropdown.tsx

```diff
@@ -19,403 +12,58 @@ import {selectValue} from 'src/variables/actions/thunks'
-        this.handleSelect(shownValues[selectIndex], true)
-      } else {
-        // the person could have been typing and pressed return, need to
-        // make sure the value in the input field is real/legal:
-
-        // but:  if the value they typed is LEGAL (in the list/dropdown values), set it;
-        // else: reset to the previous real/legal value:
-        if (values.includes(typedValue)) {
-          // is a real legal value
-          this.handleSelect(typedValue, true)
-        } else {
-          const newState = {
-            menuOpen: MenuStatus.Closed,
-            selectIndex: -1,
-            ...this.getRealValue(),
-          }
-          this.setState(newState)
-        }
-      }
-    }
-  }
-
-  // want on click away to set it? no
-  // if the entire item loses focus, then
-  // the input should show the actual selected item, not what the user typed in;
-  // only want to show valid values when the component is not actively being used
-  onClickAwayHere = () => {
-    //  reset:
-    this.setState(this.getRealValue())
-  }
-
-  getRealValue = () => {
-    const {actualVal, typedValue} = this.state
-    const {selectedValue} = this.props
-
-    if (actualVal || selectedValue) {
-      const realValue = selectedValue ?? actualVal
-
-      if (typedValue !== realValue) {
-        return {typedValue: realValue}
-      }
-    }
   }
 
   render() {
-    const {selectedValue, values, name, status} = this.props
-    const {typedValue, shownValues, menuOpen, selectIndex} = this.state
-    const dropdownStatus =
-      values.length === 0 ? ComponentStatus.Disabled : ComponentStatus.Default
-
-    const placeHolderText = this.getPlaceHolderText('Select a Value')
-
-    const widthStyle = this.getWidth(placeHolderText)
-
-    const selectAllTextInInput = (event?: ChangeEvent<HTMLInputElement>) => {
-      if (event) {
-        event.target.select()
-      }
-    }
-
-    const getInnerComponent = () => {
-      if (
-        status === RemoteDataState.Loading ||
-        status === RemoteDataState.Error ||
-        this.noValuesPresent()
-      ) {
-        return placeHolderText
-      } else {
-        return (
-          <Input
-            placeholder={placeHolderText}
-            onChange={this.filterVals}
-            value={typedValue}
-            onKeyDown={this.maybeSelectNextItem}
-            testID={`variable-dropdown-input-typeAhead--${name}`}
-            onFocus={selectAllTextInInput}
-          />
-        )
-      }
-    }
-
+    const {selectedValue, values, name} = this.props
+    const typeAheadItems = values.map(val => ({
+      id: val,
+      name: val,
+    })) as SelectableItem[]
+    const selectedVariableValue = {
+      id: selectedValue,
+      name: selectedValue,
+    } as SelectableItem
     return (
-      <Dropdown
-        style={{width: '192px'}}
-        className="variable-dropdown--dropdown"
+      <TypeAheadDropDown
+        style={{width: '150px'}}
```

Comment:
If this style needs to be inline, rather than part of a stylesheet, can you move the definition of `{width: '150px'}` to outside of the component and use the variable here please?